archibald/cogs/alias.py

```diff
@@ -0,0 +1,57 @@
+import discord
+import json
+from discord import option
+from discord.ext import commands
+
+class link_alias(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.link_map = json.loads(open("archibald/cogs/links.json").read())['links'];
+        self.add_alias_whitelist = [
+            332935845004705793, 
+            655144378062471173, 
+            845784118536175676, 
+            464247726381727744, 
+            192732093019717632  
+        ]
+
+    @commands.Cog.listener()
+    async def on_message(self, message: discord.Message):
```

Comment:
Let's use a command that's something like `-link cshub` and `cshub` is a string arg